exoscale/datasource_exoscale_compute.go

```diff
@@ -0,0 +1,244 @@
+package exoscale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/exoscale/egoscale"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCompute() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:          schema.TypeString,
+				Description:   "ID of the Compute",
```

Comment:
Even though it makes sense to keep the DS name `exoscale_compute` to match its equivalent resource, the actual resource is a Compute *instance* so you should fix all the descriptions and the documentation website content to reflect this.

```suggestion
				Description:   "ID of the Compute instance",
```